packages/lib/date-ranges.test.ts

```diff
@@ -48,33 +47,32 @@ describe("processWorkingHours", () => {
 
     expect(lastAvailableSlot.start.date()).toBe(31);
   });
-  // TEMPORAIRLY SKIPPING THIS TEST - Started failing after 29th Oct
-  it.skip("should return the correct working hours in the month were DST ends", () => {
+  it("should return the correct working hours in the month were DST ends", () => {
     const item = {
       days: [0, 1, 2, 3, 4, 5, 6], // Monday to Sunday
-      startTime: new Date(Date.UTC(2023, 5, 12, 8, 0)), // 8 AM
-      endTime: new Date(Date.UTC(2023, 5, 12, 17, 0)), // 5 PM
+      startTime: new Date(2023, 5, 12, 8, 0), // 8 AM
+      endTime: new Date(2023, 5, 12, 17, 0), // 5 PM
     };
 
     // in America/New_York DST ends on first Sunday of November
     const timeZone = "America/New_York";
 
-    let firstSundayOfNovember = dayjs().startOf("day").month(10).date(1);
+    let firstSundayOfNovember = dayjs.utc().startOf("day").month(10).date(1);
     while (firstSundayOfNovember.day() !== 0) {
       firstSundayOfNovember = firstSundayOfNovember.add(1, "day");
     }
 
-    const dateFrom = dayjs().month(10).date(1).startOf("day");
-    const dateTo = dayjs().month(10).endOf("month");
+    const dateFrom = dayjs.utc().month(10).date(1).startOf("day");
+    const dateTo = dayjs.utc().month(10).endOf("month");
 
     const results = processWorkingHours({ item, timeZone, dateFrom, dateTo });
 
     const allDSTStartAt12 = results
       .filter((res) => res.start.isBefore(firstSundayOfNovember))
-      .every((result) => result.start.utc().hour() === 12);
+      .every((result) => result.start.utc().hour() === 11);
```

Comment:
Here the problem was that the hour would change based on the system time, fixed it by ensuring the firstSundayOfNovember variable is defined in UTC. Technically 11,12 would work right now as well - but this would break on the next DST change.